kiali-server/templates/service.yaml

```diff
@@ -34,11 +34,17 @@ spec:
   {{- end }}
     protocol: TCP
     port: {{ .Values.server.port }}
+    {{- if eq .Values.deployment.service_type "nodePort" }}
+    nodePort: {{ .Values.server.nodePort }}
+    {{- end }}
   {{- if .Values.server.observability.metrics.enabled }}
   - name: http-metrics
     appProtocol: http
     protocol: TCP
     port: {{ .Values.server.observability.metrics.port }}
+    {{- if eq .Values.deployment.service_type "nodePort" }}
```

Comment:
same as above - check for equality to `NodePort` (capital "N").

Reply:
Fixed. Thanks!